sabitou_clients/lib/screens/profile/profile_view.dart

```diff
@@ -0,0 +1,755 @@
+                  color: Colors.green,
+                  fontWeight: FontWeight.bold,
+                ),
+                textAlign: TextAlign.center,
+              ),
+            ),
+            const SizedBox(width: 16.0),
+            if (isAdmin)
+              ElevatedButton(
+                onPressed: () {},
+                style: ElevatedButton.styleFrom(
+                  padding: const EdgeInsets.symmetric(
+                    vertical: 16.0,
+                    horizontal: 24.0,
+                  ),
+                ),
+                child: Text(
+                  AppInternationalizationService.to.changeTheBusiness,
+                  style: const TextStyle(
+                    fontWeight: FontWeight.bold,
+                  ),
+                  textAlign: TextAlign.center,
+                ),
+              ),
+          ],
+        ),
+      ],
+    );
+  }
+}
+
+class _StoreSection extends StatelessWidget {
+  final String store;
+  final bool isAdmin;
+  const _StoreSection({
+    required this.store,
+    required this.isAdmin,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        Text(
+          AppInternationalizationService.to.store,
+          style: const TextStyle(fontWeight: FontWeight.bold),
+          textAlign: TextAlign.center,
+        ),
+        Text(AppInternationalizationService.to.manageTheStore),
+        const SizedBox(height: 5.0),
+        Wrap(
+          runSpacing: 16.0,
+          spacing: 16.0,
+          children: [
+            OutlinedButton(
+              onPressed: () {},
+              style: OutlinedButton.styleFrom(
+                side: const BorderSide(color: Colors.green, width: 2.0),
+                padding: const EdgeInsets.symmetric(
+                  vertical: 16.0,
+                  horizontal: 24.0,
+                ),
+              ),
+              child: Text(
+                store,
+                style: const TextStyle(
+                  color: Colors.green,
+                  fontWeight: FontWeight.bold,
+                ),
+                textAlign: TextAlign.center,
+              ),
+            ),
+            const SizedBox(width: 16.0),
+            if (isAdmin)
+              ElevatedButton(
+                onPressed: () {},
+                style: ElevatedButton.styleFrom(
+                  padding: const EdgeInsets.symmetric(
+                    vertical: 16.0,
+                    horizontal: 24.0,
+                  ),
+                ),
+                child: Text(
+                  AppInternationalizationService.to.changeTheStore,
+                  style: const TextStyle(
+                    fontWeight: FontWeight.bold,
+                  ),
+                  textAlign: TextAlign.center,
+                ),
+              ),
+          ],
+        ),
+      ],
+    );
+  }
+}
+
+/// Temporal mock user data.
+
+class User {
```

Comment:
This model is not yet useful,  you can directly import the one from grpc or userPreference. like is down on dashboard.